packages/itwinui-css/src/button/base.scss

```diff
@@ -19,6 +19,7 @@ $disabledButtonSelectors: '[disabled], :disabled, [aria-disabled="true"], [data-
   padding-block: var(--_iui-button-padding-block);
   padding-inline: var(--_iui-button-padding-inline);
   text-decoration: none;
+  touch-action: manipulation;
```

Comment:
Maybe this should go in `iui-button-base`?

One thing I would like to verify is that it doesn't break links used as buttons, e.g. `<a class="iui-button-base iui-button">`

Reply:
Moved to `.iui-button-base` and tested with the breadcrumb anchors.

I'm wondering if this should actually be moved into `.iui-field` to be applied to `<input>` and `<textarea>` as well. 🤔

Reply:
I think it's fine in `iui-button-base`. Input/textarea likely have their own double tap behaviors (e.g. select text)